### DISCUSSION TITLE
💰 Pricing Query for Pro plan and OpenSource projects

### DISCUSSION THREAD

**Discussion by jd-solanki** [vercel/community#5887] [Account Issue] [answered]
### Question

Hi 👋🏻

I would like to go for paid plan and I have query regarding how usage will get calculated and reflected in my bill.

I few open-source projects and few private projects, If I purchase the pro plan will my open-source project usage will get calculated in my pro plan bill? For example, if my open-source project hosting uses 300GB 
 bandwidth and my private project hosting uses 800GB will my bill get generated for 1100GB (1TB+ which is over limit) or open-source project usage will be ignore and I'll get billed for 800GB (under 1TB limit)?

Awaiting your response.

Thanks,
ThemeSelection Team

**Comment by amyegan** [comment, marked as answer]
Hi @jd-solanki. All [usage](https://vercel.com/docs/limits/usage) is counted the same on a Pro plan regardless of whether the project is open-source, private, or simply in a public repo. Sponsorships are sometimes offered, and we have supported over 700 teams and individuals with Vercel credits for open-source projects and non-profits, but we aren't accepting _new_ team sponsorships at the moment.

https://vercel.com/guides/can-vercel-sponsor-my-open-source-project
